python/README.md: fix note about python[3]-codecs

commodo opened the pull request:
Maintainer: @jefferyto , me
Compile tested: N/A
Run tested: N/A

------------------------------------------------------------

The note was note correct when mentioning encodings vs python[3]-codecs.
This change fixes this confusion.

Signed-off-by: Alexandru Ardelean <ardeleanalex@gmail.com>

commodo commented:
took me a while to write this

jefferyto commented:
Appreciate the effort :+1: 

I don't want to diminish the important of the explanation, but the technical reasons why python3-codecs isn't needed as a dependency for python3-lxml isn't "core" to the point of that section (that the makefile author should check the `DEPENDS` info for both variants)... Perhaps the full explanation can be moved to a footnote/end note, so it doesn't distract the reader from the main point?

commodo commented:
@jefferyto 
added footnote [more or less] ; if you know some smarter markdown stuff [that works with Github] I'm open

otherwise:
Changelog v1 -> v2:
* moved explanation to footnote

jefferyto commented:
Would in-page anchors/links work in GitHub (like https://stackoverflow.com/a/29384216)?

commodo commented:
I’ll check.
I’m on vacation this week.
So, it’ll take a while.

I also found those but wasn’t sure about the HTML mixin.

jefferyto commented:
This isn't urgent - enjoy your vacation :joy:

commodo commented:
@jefferyto 

changelog v1 -> v2:
* did a few push-forces
* rebased on newer master
* added click-able anchor/footnote for encodings;

if i missed anything or if something should be updated/changed feel free to mention

thanks
Alex

jefferyto commented:
LGTM - thanks!
